extensions/amp-story-shopping/0.1/amp-story-shopping-tag.js

```diff
@@ -85,13 +92,17 @@ export class AmpStoryShoppingTag extends AMP.BaseElement {
     });
   }
 
-  /**
-   * @private
-   */
+  /** @private */
   onClick_() {
     this.storeService_.dispatch(Action.ADD_SHOPPING_DATA, {
       'activeProductData': this.tagData_,
     });
+    this.openPageAttachment_();
+  }
+
+  /** @private */
+  openPageAttachment_() {
+    this.pageAttachmentEl_.getImpl().then((impl) => impl.open());
```

Comment:
Would it be possible to click on the tag when the page attachment extension hasn't loaded yet? If so, `whenUpgradedToCustomElement` would make sense to do here (otherwise you could see the error "pageAttachmentEl_ has no exported member getImpl" or similar)